Description: Point to pysolfc instead of pysol.py, and remove Encoding=UTF-8
 -- see http://standards.freedesktop.org/desktop-entry-spec/1.0/apc.html
Author: Bernhard Reiter <ockham@raz.or.at>
Forwarded: https://sourceforge.net/tracker/?func=detail&aid=2946832&group_id=150718&atid=778745

--- a/data/pysol.desktop
+++ b/data/pysol.desktop
@@ -1,8 +1,8 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=PySol Fan Club Edition
-Exec=pysol.py
+Exec=pysolfc
 Terminal=false
 Type=Application
 Categories=Game;CardGame;
+Keywords=Game;CardGame;
 Icon=/usr/share/icons/pysol01.png
